### PR TITLE
zeroex - update eth native taker

### PR DESCRIPTION
**Pull request by RantumBits**
zeroex - update taker on native swaps

### CODE REVIEW

File: dex/models/_projects/zeroex/ethereum/zeroex_ethereum_native_fills.sql [hunk 1]
```diff
@@ -256,25 +256,13 @@ WITH
       FROM {{ source('zeroex_ethereum', 'ExchangeProxy_evt_RfqOrderFilled') }} fills
       LEFT JOIN {{ source('prices', 'usd') }} tp ON
           date_trunc('minute', evt_block_time) = tp.minute
-          {% if is_incremental() %}
-          AND {{ incremental_predicate('tp.minute') }}
-          {% endif %}
-          {% if not is_incremental() %}
-          AND evt_block_time >= TIMESTAMP '{{zeroex_v3_start_date}}'
-          {% endif %}
```
Comment (Hosuke) [inline dex/models/_projects/zeroex/ethereum/zeroex_ethereum_native_fills.sql:264]: The incremental filters are removed for prices join, but this change may have minor impact.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RantumBits

File: dex/models/_projects/zeroex/ethereum/zeroex_ethereum_native_fills.sql [hunk 2]
```diff
@@ -256,25 +256,13 @@ WITH
       FROM {{ source('zeroex_ethereum', 'ExchangeProxy_evt_RfqOrderFilled') }} fills
       LEFT JOIN {{ source('prices', 'usd') }} tp ON
           date_trunc('minute', evt_block_time) = tp.minute
-          {% if is_incremental() %}
```
Comment (aalan3) [inline dex/models/_projects/zeroex/ethereum/zeroex_ethereum_native_fills.sql:259]: Why remove this? This will make the runtime much slower.

**Review by aalan3 — CHANGES_REQUESTED** [review]

File: dex/models/_projects/zeroex/ethereum/zeroex_ethereum_native_fills.sql (hunk 2 above)
Comment (RantumBits) [reply dex/models/_projects/zeroex/ethereum/zeroex_ethereum_native_fills.sql:259]: np, added back

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**